artview/components/roi.py

```diff
@@ -192,16 +192,30 @@ def CreateROIWidget(self):
         self.buttonOpenTable = QtGui.QPushButton('Open Tabular Data', self)
         self.buttonSaveTable = QtGui.QPushButton('Save Tabular Data', self)
         self.buttonResetROI = QtGui.QPushButton('Reset ROI', self)
+        self.buttonHelp = QtGui.QPushButton('Help', self)
         self.buttonViewTable.clicked.connect(self.viewTable)
         self.buttonOpenTable.clicked.connect(self.openTable)
         self.buttonSaveTable.clicked.connect(self.saveTable)
         self.buttonResetROI.clicked.connect(self.resetROI)
+        self.buttonHelp.clicked.connect(self.displayHelp)
 
         # Create functionality buttons
+
         self.rBox_layout.addWidget(self.buttonViewTable)
         self.rBox_layout.addWidget(self.buttonOpenTable)
         self.rBox_layout.addWidget(self.buttonSaveTable)
         self.rBox_layout.addWidget(self.buttonResetROI)
+        self.rBox_layout.addWidget(self.buttonHelp)
+
+    def displayHelp(self):
+        # XXX I invite anyone to improve this help information
+        text = "Draw a Path in the Display Window using the Mouse\n\n"
+        text += "Functions:\n"
+        text += "    Primary Mouse Button - add vertex\n"
+        text += "    Hold - freely draw path\n"
+        text += "    Secundary Button - close path\n"
```

Comment:
by holding one can draw a curve, not just a polygon

I'm totally open for wording, your is much nicer than mine. One Idea I had is that roi can still use some doc-strings in how it works, and we could use the doc-string itself, but that was just an idea, do what you think is better.